lib/server/controllers/metis_controller.rb

```diff
@@ -29,5 +29,23 @@ def require_folder(bucket, folder_path)
 
       return folder
     end
+
+    def get_file_obj_from_path(path)
```

Comment:
Could these move into Metis::Path itself?

Reply:
So one reason I left those here is that they also implicitly do some validation checks as part of the `require_bucket` call. Let me see how I can keep that behavior, especially for `copy` (I think we explicitly do the checks in `bulk_copy` already).